crt_portal/api/filters.py

```diff
@@ -88,11 +88,12 @@ def form_letters_filter(querydict):
     form_letters_payload["total_form_letters"] = len(filtered_form_letters)
 
     for form_letter in filtered_form_letters:
-        email_title = form_letter.description.split("'")[1]
-        try:
-            form_letters_counter[email_title] += 1
-        except KeyError:
-            form_letters_counter[email_title] = 1
+        if "'" in form_letter.description:
```

Comment:
What happens if there ins't a `'` in the email_title? Does form_letter_payload receive a null value since form_letters_counter doesn't have a value associated with it?

Reply:
@billyhunt is there always a string in the form_letter.description? I don't know how critical additional exception handling is for this situation.

Reply:
That is set in the actions model, so we should be good there.